guides/common/modules/proc_creating-a-composite-content-view.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Creating_a_Composite_Content_View_{context}"]
-= Creating a Composite Content View
+= Creating a composite content view
 
 Use this procedure to create a composite Content View.
```

Comment:
Is "Content View" always capitalized?

Reply:
Per  https://github.com/theforeman/foreman-documentation/issues/2553, we seem to want to use lowercase "content view". But since this PR only changes headings, it might be best to keep CVs capitalized in headings for now..? Just to ensure consistency with non-heading occurrences of CVs.

Reply:
OK. So Avital's diff is correct IMHO. We can probably do a quick follow-up PR to change all other instances.

Reply:
+1 to fixing "Content View" in separate PR.